test/unit/auth/auth-api-request.spec.ts

```diff
@@ -534,6 +564,75 @@ describe('FirebaseAuthRequestHandler', () => {
     });
   });
 
+  describe('getAccountInfoByPhoneNumber', () => {
+    const httpMethod = 'POST';
+    const host = 'www.googleapis.com';
+    const port = 443;
+    const path = '/identitytoolkit/v3/relyingparty/getAccountInfo';
+    const timeout = 10000;
+    it('should be fulfilled given a valid phoneNumber', () => {
+      const expectedResult = {
+        users : [
+          {phoneNumber: '+11234567890'},
```

Comment:
I added a more realistic response and I test the expected response is returned from this operation which is enough for this API. This API returns a response that resolves with the same server response. UserRecord parses that content. I have tests for that in that file. auth.ts passes the response to UserRecord. I have a check for the whole flow in auth.spec.ts. I stub the response of getAccountInfoByPhoneNumber and then check that it is parsed correctly into the expected UserRecord.